main.tf

```diff
@@ -9,14 +9,7 @@ resource "aws_opensearch_domain" "os" {
   for_each              = var.create_os ? var.os_group : {}
   domain_name           = "${each.value.domain_name}-${var.env_name}"
   engine_version        = each.value.engine_version
-
-#  # advanced_options = var.advanced_options
-#domain_endpoint_options {
```

Comment:
I think we do need certificate and custom endpoint actually (could be for version2) 
something like this:
custom_endpoint = "${each.value.domain_name}.${replace(var.env_name,"-","")}.tolunainsights-internal.com"